### PR TITLE
Overhaul the wayfinding guide

### PR DESCRIPTION
**Pull request by kytrinyx**
This fixes broken links, and also updates some sections based on new
developments in the projects.

I've extracted all the links into link references, reducing duplication
and making it easier to fix or update links going forward.

### CODE REVIEW

File: finding-your-way.md [hunk 1]
```diff
@@ -71,37 +72,20 @@
...
-- user's profile page: [`exercism/discussions?profile`](https://github.com/exercism/discussions/issues?q=is%3Aissue+is%3Aopen+profile+sort%3Aupdated-desc) / [`exercism/exercism.io?profile`](https://github.com/exercism/exercism.io/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20profile%20+sort%3Aupdated-desc),
-- user's account page: [`exercism/discussions?account`](https://github.com/exercism/discussions/issues?q=is%3Aissue+is%3Aopen+account+sort%3Aupdated-desc) / [`exercism/exercism.io?account`](https://github.com/exercism/exercism.io/issues?q=is%3Aissue+is%3Aopen+account+sort%3Aupdated-desc).
-
-If the idea doesn't fit in one of those discussions, then if it relates to:
-
-* a specific part of the site, start a new conversation in the [`exercism/exercism.io` repository](https://github.com/exercism/exercism.io/issues/new).
-- multiple aspects of the site, kick it off in the [`exercism/discussions` repository](https://github.com/exercism/discussions/issues/new).
+We're currently working with professional user experience designers on rethinking the entire experience from scratch. Check out the [roadmap][] for more details about this.
```
Comment (Insti) [inline finding-your-way.md:75]: Roadmap should probably link somewhere?

Comment (kytrinyx) [reply finding-your-way.md:75]: It does. That's the markdown syntax equivalent to `[roadmap][roadmap]`, and there's a reference url at the bottom of the document. 

You can view the rendered document here:
https://github.com/exercism/docs/blob/overhaul-wayfinding-guide/finding-your-way.md

Comment (Insti) [reply finding-your-way.md:75]: Ok. 👍 for fancy markdown syntax.

**Review by Insti — APPROVED** [review]
Looks OK to me, I've not checked every link, but I didn't catch anything horribly out of place during my read-through.

**Comment by kytrinyx** [comment]
Thanks for taking the time to review it @Insti!

**Pull request merged**